### PR TITLE
Add voice commands for the bag actions

### LINKED ISSUE #4: Add all voice commands for the bag
- [x] Create shortcuts for the "potion" commands
- [x] Skip confirm

### PR DESCRIPTION
**Pull request by AdamJonsson**
Currently, the voice commands only work on the first page of the bag
Closes #4

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -28,43 +32,78 @@ def main():
         try: 
             if intent == Intents.GO:
                 pressReleaseKey(NoGbaMappings.A)
+
             elif intent == Intents.BACK:
                 pressReleaseKey(NoGbaMappings.B)
+                bagActions.markBagAsClosed()
```
Comment (ellaklara) [inline main.py]: Instance of 'BagActions' has no 'markBagAsClosed' member :(
I don't know what to replace it with

**Review by ellaklara — CHANGES_REQUESTED** [review]
Looks nice, I just found a small error!
Right now it doesn't work the second time you try to use a potion, but that might be because of the fact that markBagAsClosed doesn't work?

File: main.py (hunk 1 above)
Comment (AdamJonsson) [reply main.py]: Ohhh, forgot to change that

**Comment by AdamJonsson** [comment]
Seems like our current quota has been used up for today 🛑

**Comment by ellaklara** [comment]
Haha, I'll look at it tomorrow then!

**Review by ellaklara — APPROVED** [review]
Works like a charm now!

**Pull request merged**